sass/elements/site-banner.scss

```diff
@@ -84,7 +84,7 @@ polyfill-next-selector { content: ':host header.shrink'; }
 polyfill-next-selector { content: 'header summary'; }
 :host ::content header summary {
   margin-top: 20px;
-  opacity: 0.6;
+  font-size: 20px;
```

Comment:
The a11y concern here is that with the opacity: 0.6, the contrast is really, really terrible. I had actually bumped this up to 24px after feedback from @lpalmaro. I can add screenshots if you like.